common/primitives/src/schema.rs

```diff
@@ -120,54 +215,110 @@ pub struct SchemaInfoResponse {
 	pub schema_id: SchemaId,
 	/// The model format type for how the schema model is represented
 	pub model_type: ModelType,
-	/// The payload location
+	/// The payload location associated with this Schema's associated Intent
 	pub payload_location: PayloadLocation,
-	/// grants for the schema
-	pub settings: Vec<SchemaSetting>,
+	/// The settings for this Schema's associated Intent
+	pub settings: Vec<IntentSetting>,
+	/// The IntentId of the Intent that this Schema implements
+	pub intent_id: IntentId,
+	/// The status of this Schema
+	pub status: SchemaStatus,
```

Comment:
nit: truism - because "status" is a general term, this comment doesn't help a reader see the purpose of the the field.